### PR TITLE
DE: bad de link

### PR DESCRIPTION
**Pull request by rshorey**
one bad link. surprisingly painful to fix. delaware.

### CODE REVIEW

**Comment by paultag** [comment]
Doesn't this add 2x overhead to the normal scrape? Since you have to HEAD and GET every page?

File: openstates/de/bills.py [hunk 1]
```diff
@@ -50,8 +50,13 @@ def scrape_bill(self,link,chamber,session):
 
         base_url = "http://legis.delaware.gov"
         text_base_url = "http://legis.delaware.gov/LIS/lis{session}.nsf/vwLegislation/{bill_id}/$file/legis.html?open"
-        page = self.lxmlize(link)
+
```
Comment (paultag) [inline openstates/de/bills.py]: mind triming whitespace out?

**Comment by rshorey** [comment]
@paultag here's another way that doesn't increase overhead but puts lxmlize in a try which I don't love, but is probably better.

**Comment by paultag** [comment]
Yeah, slightly more ugly, but would reduce network overhead by half, so I'm cool with it :)

**Comment by paultag** [comment]
Thanks!

**Pull request merged**